app/views/users/show.html.erb

```diff
@@ -1 +1,12 @@
-<h1>Here is the profile of a user including user Bio and user posts</h1>
\ No newline at end of file
+<h2><%= @user.name%>'s Most Recent Posts </h2>
+
+<%= render partial: "shared/user/user_header", locals: { user: @user } %>
+<%= render partial: "shared/user/user_bio", locals: { user: @user } %>
+
+<% @recent_posts.order(comments_counter: :DESC).each do |post| %>
+  <%= render partial: "shared/post/post_card", locals: { post: post } %>
+<% end %>
+
```

Comment:
- [ ] In the user's show page, you just have to show the number of comments and likes and not the details of the comments.

### User's show page (Given in wireframe)
![Screenshot from 2022-07-13 16-07-11](https://user-images.githubusercontent.com/26767383/178714617-c45b70f8-aa6f-4c83-ae25-9c55694e7a96.png)



### Your user's show page

![Screenshot from 2022-07-13 16-04-01](https://user-images.githubusercontent.com/26767383/178714548-22e40e38-345e-4198-8f65-a8b9026aac2a.png)

Reply:
Hello @Deepakdanger thanks, I'll make that change